definitions/infra-awselasticbeanstalkenvironment/summary_metrics.yml

```diff
@@ -1,8 +1,3 @@
-providerAccountName:
```

Comment:
What is the reason to remove this one? I think is provided in pretty much all AWS summaries as you can see [here](https://github.com/newrelic-experimental/entity-synthesis-definitions/search?q=providerAccountName)